src/common/quantile.h

```diff
@@ -651,6 +651,9 @@ class QuantileSketchTemplate {
      * \param src the source summary
      * \param max_nbyte maximum number of byte allowed in here
      */
+#ifndef _WIN32
+    #pragma omp declare simd uniform(src, max_nbyte)
```

Comment:
Why is this windows specific?

Reply:
in short visual studio don't support omp 4.5, this is legacy code. Will remove this in favor of compiler hinted approach. 

cc  @SmirnovEgorRu

Reply:
removed.